failover/README.md

```diff
@@ -0,0 +1,107 @@
+# MySQL Failover Demo
+
+This is a showcase for MySQL deployment in clustered configuration and demonstration of Debezium execution after a database failure.
+
+## Topology
+
+The deployment consists of the following components
+
+* Database
+  * MySQL 1 instance (configured as a slave to MySQL 2) with GTID enabled
+  * MySQL 2 instance (configured as a slave to MySQL 1) with GTID enabled
+  * HAProxy instance - MySQL 1 is configured as the primary server, MySQL 2 as a backup
+* Streaming system
+  * Apache ZooKeeper
+  * Apache Kafka broker
+  * Apache Kafka Connect with Debezium MySQL Connector - the connector will connect to HAProxy
+
+## Demonstration
+
+Start the components and register Debezium to stream changes from the database
+```
+export DEBEZIUM_VERSION=0.9
+docker-compose -f docker-compose-mysql.yaml up --build
```

Comment:
I'd suggest to rename the file to _docker-compose.yaml_, allowing to omit the `-f` option.

Reply:
@gunnarmorling Right, changed